ingest/src/settings.rs

```diff
@@ -16,6 +17,8 @@ pub struct Settings {
     pub listen: String,
     /// Local folder for storing intermediate files
     pub cache: String,
+    /// Network required in all public keys:  mainnet | testnet
+    pub network: String,
```

Comment:
You should be able to use the Network type instead of a String

Reply:
Fixed 075fa20